<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalcView</class>
 <widget class="QWidget" name="CreditCalcView">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>615</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>541</width>
    <height>615</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>541</width>
    <height>615</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sans-serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Credit Calculator</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #191A19;
	color: white;
	font-family: Sans-serif;
}

QLineEdit {
	border: 1px solid #606060;
	border-radius: 25px;
	padding-left: 10px;
}

QTextBrowser {
	padding-left: 10px;
}

QPushButton {
	border: 1px solid #606060;
	border-radius: 25px;
}

QPushButton::hover {
	background-color: #444444;
}

QPushButton::pressed {
	background-color: #282828;
}

QSpinBox {
	border: 1px solid #606060;
	border-radius: 25px;
	padding-left: 10px;
}

QDoubleSpinBox {
	border: 1px solid #606060;
	border-radius: 25px;
	padding-left: 10px;
}

QComboBox {
	border: 1px solid #606060;
	border-radius: 25px;
	padding-left: 10px;
}
QComboBox::item::selected {
	background-color: #444444;
}</string>
  </property>
  <widget class="QPushButton" name="homeBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #374045;
}

QPushButton::hover {
	background-color: #3A4750;
}

QPushButton::pressed {
	background-color: #303841;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="equalBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #E16428;
}

QPushButton::hover {
	background-color: #ff8000;
}

QPushButton::pressed {
	background-color: #b35900;
}</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="termLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Interest rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="typeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Payment type</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="amountValueSpin">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>301</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	border-top-right-radius:0;
	border-bottom-right-radius:0;
}</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>10000000000.000000000000000</double>
   </property>
   <property name="value">
    <double>100000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="amountLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Credit amount</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="termValueSpin">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>191</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-right-radius: 0;
border-bottom-right-radius: 0;
</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="amountLabel_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Credit term</string>
   </property>
  </widget>
  <widget class="QComboBox" name="termTypeComboBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>111</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:0;</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>years</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="minimumContentsLength">
    <number>0</number>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>years</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>months</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="typeValueWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-right-radius:0;
border-bottom-right-radius:0;</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>Annuity</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
   </property>
   <property name="minimumContentsLength">
    <number>0</number>
   </property>
   <property name="placeholderText">
    <string>Annuity</string>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Annuity</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Differentiated</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="rateValueSpin">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>301</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	border-top-right-radius:0;
	border-bottom-right-radius:0;
}</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>999.990000000000009</double>
   </property>
   <property name="value">
    <double>20.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="tableBtn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>550</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #414141;</string>
   </property>
   <property name="text">
    <string>Payment schedule</string>
   </property>
  </widget>
  <widget class="QChartView" name="chartView">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>260</y>
     <width>421</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLabel" name="rubLabel_1">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>60</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RUB</string>
   </property>
  </widget>
  <widget class="QLabel" name="rubLabel_2">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>130</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RUB</string>
   </property>
  </widget>
  <widget class="QLabel" name="rubLabel_3">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>190</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RUB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #1A1A2E;
}

QPushButton::hover {
	background-color: #0F3460;
}

QPushButton::pressed {
	background-color: #0A043C;
}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="iconLabel_1">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>400</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: #F05454;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="iconLabel_2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>360</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #03506F;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="amountLabel_3">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>360</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Credit</string>
   </property>
  </widget>
  <widget class="QLabel" name="amountLabel_4">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>400</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Overpayment</string>
   </property>
  </widget>
  <widget class="QLabel" name="rubLabel_4">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>250</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RUB</string>
   </property>
  </widget>
  <widget class="QLabel" name="lastPaymentLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>250</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Last  Payment</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="lastPayAnswer">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>250</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>1000000000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="overpaymentLabel">
   <property name="geometry">
    <rect>
     <x>551</x>
     <y>131</y>
     <width>144</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Overpayment</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalpayLabel">
   <property name="geometry">
    <rect>
     <x>551</x>
     <y>189</y>
     <width>129</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total payout</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="monthPayAnswer">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>60</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>1000000000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="totalPayAnswer">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>190</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>1000000000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="overPayAnswer">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>130</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>1000000000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="monthPayLabel">
   <property name="geometry">
    <rect>
     <x>551</x>
     <y>63</y>
     <width>183</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Monthly Payment</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
